docs/trace/customizing-the-sdk/README.md

```diff
@@ -193,6 +193,11 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
 tracerProvider.AddProcessor(new MyProcessor3());
 ```
 
+**Note:** The order of processor registration is important. Each processor added
```

Comment:
Maybe switch all the notes to `> **Note**`:

![image](https://user-images.githubusercontent.com/17327289/215882036-752da09d-b387-4bf4-8ad5-e9739630a703.png)

Reply:
I had the same thought but I didn't want to pollute this PR with a bunch of random stuff. I'll do a separate PR for updating the notes across the whole repo.